src/scenes/gameuiscreen.ts

```diff
@@ -100,6 +113,69 @@ export class GameUiScreen extends Phaser.Scene {
         );
     }
 
+    public displayProfiles(): void {
+        const allPlayers: Nullable<Player[]> = PlayerManager.getAll();
```

Comment:
Define a function in PlayerManager that returns the total number of players? I need it in GameUiScreen as well